crates/burn-fusion/src/stream/context.rs

```diff
@@ -630,6 +630,16 @@ impl RelativeOps for BoolOperationIr {
                 input: desc.input.to_relative(converter),
                 out: desc.out.to_relative(converter),
             }),
+            BoolOperationIr::And(desc) => BoolOperationIr::And(BinaryOpIr {
+                lhs: desc.lhs.to_relative(converter),
+                rhs: desc.rhs.to_relative(converter),
+                out: desc.out.to_relative(converter),
+            }),
+            BoolOperationIr::Or(desc) => BoolOperationIr::And(BinaryOpIr {
```

Comment:
Typo or copy/pasta mistake, didn't notice before
```suggestion
            BoolOperationIr::Or(desc) => BoolOperationIr::Or(BinaryOpIr {
```